File: Dia 2/Python/exercicio1.py
def main():
    vetor = [0] * 8

    for i in range(8):
        vetor[i] = int(input(f"Digite o valor da posição {i}: "))

    x = int(input("Digite o valor de X: "))
    y = int(input("Digite o valor de Y: "))

    if x < 0 or x >= len(vetor):
        print("Valor de X fora dos limites do vetor.")
        return

    if y < 0 or y >= len(vetor):
        print("Valor de Y fora dos limites do vetor.")
        return

    soma = vetor[x] + vetor[y]

    print(f"A soma dos valores nas posições {x} e {y} é {soma}.")

if __name__ == "__main__":
    main()

File: Dia 2/Go Lang/exercicio1.go
package main

import (
	"fmt"
)

func main() {
	var vetor [8]int

	fmt.Println("Digite 8 valores para o vetor:")
	for i := 0; i < 8; i++ {
		fmt.Printf("Digite o valor para a posição %d: ", i)
		fmt.Scan(&vetor[i])
	}

	var x, y int
	fmt.Print("Digite o valor de X: ")
	fmt.Scan(&x)
	fmt.Print("Digite o valor de Y: ")
	fmt.Scan(&y)

	if x >= 0 && x < 8 && y >= 0 && y < 8 {
		soma := vetor[x] + vetor[y]
		fmt.Printf("A soma dos valores nas posições %d e %d é %d\n", x, y, soma)
	} else {
		fmt.Println("Índices X e Y devem estar entre 0 e 7.")
	}
}
